Cargo.toml

```diff
@@ -97,32 +97,32 @@ rustls = { version = "0.21.12", features = ["dangerous_configuration"] }
 rustls-pemfile = "1.0.4"
 script_layout_interface = { path = "components/shared/script_layout" }
 script_traits = { path = "components/shared/script" }
-selectors = { git = "https://github.com/servo/stylo", branch = "2024-07-16" }
+selectors = { git = "https://github.com/servo/stylo", rev = "refs/pull/65/head" }
 serde = "1.0.208"
 serde_bytes = "0.11"
 serde_json = "1.0"
 servo-media = { git = "https://github.com/servo/media" }
 servo-media-dummy = { git = "https://github.com/servo/media" }
 servo-media-gstreamer = { git = "https://github.com/servo/media" }
-servo_arc = { git = "https://github.com/servo/stylo", branch = "2024-07-16", features = ["servo"] }
-servo_atoms = { git = "https://github.com/servo/stylo", branch = "2024-07-16" }
-size_of_test = { git = "https://github.com/servo/stylo", branch = "2024-07-16" }
+servo_arc = { git = "https://github.com/servo/stylo", rev = "refs/pull/65/head", features = ["servo"] }
+servo_atoms = { git = "https://github.com/servo/stylo", rev = "refs/pull/65/head" }
+size_of_test = { git = "https://github.com/servo/stylo", rev = "refs/pull/65/head" }
```

Comment:
shall we try to land this stylo change now?

Reply:
Yep. It should be ready to go now.